types/src/jwks/rsa.rs

```diff
@@ -33,6 +37,20 @@ impl RSA_JWK {
         }
     }
 
+    pub fn to_poseidon_scalar(&self) -> Result<ark_bn254::Fr> {
```

Comment:
I think this should be part of the `poseidon_bn254` module, where all hashing is defined. So other applications can use it easily (e.g., proving service).

Reply:
i will refactor this later, I want to move the whole of jwks in aptos crypto.  just moving this function right now is a little annoying without the whole struct.  I think we want to do that anyway.  Left todo.